gcp-repo/metadata.p2.inf

```diff
@@ -27,7 +27,7 @@ units.1.properties.5.name=org.eclipse.equinox.p2.doc.url
 units.1.properties.5.value=https://cloud.google.com/eclipse/docs/
 units.1.provides.1.namespace=org.eclipse.equinox.p2.iu
 units.1.provides.1.name=com.google.cloud.tools.eclipse.category
-units.1.provides.1.version=1.0.0
+units.1.provides.1.version=1.0.1
```

Comment:
I don't know! I think we have to try it and maybe change the unimportant versions to 999.999.999 to indicate that they don't matter.